pkg/version/version.go

```diff
@@ -0,0 +1,26 @@
+// Package version contains the version information of the krew binary.
+package version
+
+var (
+	// GitCommit contains the git commit idenifier.
+	GitCommit string
+
+	// GitTag contains the git tag or describe output.
+	GitTag string
+)
+
+// GitCommitString should be used to retrieve GitCommit or the default value.
+func GitCommitString() string {
```

Comment:
We should either have the variable publicly accessible or have a getter function.

Does the variable injection work if a variable is unavailable?
If injection does not work sticking to a var with an empty string is more explicit than defaulting it to `"unknown"`, this seems to be the responsibility of the frontend.

Reply:
Var has to be publicly accessible otherwise the -X flag doesn’t work properly. 

This is a fairly common pattern and is safe. There’s a lot of undefined behavior around ldflags when the variable already has a value etc.

Reply:
I just tested to set not exported variables and it worked.
What do you mean by properly?

I think we still should return `""` instead `"unknown"`

Reply:
Seems like I recall incorrectly. Can't find any references to it.

I'll unexport this field and keep the method.